show/main.py

```diff
@@ -366,7 +366,7 @@ def cli():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def arp(ipaddress, iface, verbose):
     """Show IP ARP table"""
-    cmd = "nbrshow -4"
+    cmd = "arp -n"
```

Comment:
`nbrshow `is a script that consolidates relevant output with matching FDB entry and displays the port information as well. Whereas `arp `is a linux command. This is not the intention of `show arp` command.